content/en/docs/reference/command-line-tools-reference/feature-gates/scheduler-queueing-hints.md

```diff
@@ -13,6 +13,10 @@ stages:
   - stage: beta
     defaultValue: false
     fromVersion: "1.29"
+    toVersion: "1.31"
+  - stage: beta
+    defaultValue: true
+    fromVersion: "1.32"
 ---
 Enables [the scheduler's _queueing hints_ enhancement](https://github.com/kubernetes/enhancements/blob/master/keps/sig-scheduling/4247-queueinghint/README.md),
```

Comment:
For beta, avoid linking to a KEP; instead, link to a documentation page about scheduling
that mentions these hints.

It's OK for that document page to reference the KEP as additional reading, but KEPs
should not serve as primary documentation.